package.json

```
@@ -78,7 +78,12 @@
     "tinycolor2": "^1.4.2",
     "tslib": "^2.3.1",
     "y18n": "^5.0.8",
-    "zone.js": "0.11.4"
+    "zone.js": "0.11.4",
+    "@semantic-release/github": "^9.0.3",
+    "commitizen": "^4.3.0",
+    "conventional-changelog-conventionalcommits": "^6.1.0",
+    "inquirer": "^8.2.5",
+    "semantic-release": "^21.0.7"
```

```suggestion
    "zone.js": "0.11.4"
```

remove this and run `npm update` to refresh package-lock.json